### PR TITLE
[Help wanted] German translation for QLC+ 5

### PR DESCRIPTION
**Pull request by fxedel**
See #1076.

Everyone who has write access to the QLC+ repository can contribute on this PR to add German translations.

This PR is just to avoid duplicate work when several people create the same translations locally.

### CODE REVIEW

File: qmlui/qlcplus_de_DE.ts [hunk 1]
```diff
@@ -157,57 +157,57 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/fixturesfunctions/AddFunctionMenu.qml" line="46"/>
         <source>New folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Neuer Ordner</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/AddFunctionMenu.qml" line="57"/>
         <source>New Scene</source>
-        <translation type="unfinished"></translation>
+        <translation>Neue Scene</translation>
```
Comment (FloEdelmann) [inline qmlui/qlcplus_de_DE.ts]: Should be "Neue Szene".

Comment (FloEdelmann) [reply qmlui/qlcplus_de_DE.ts]: Done.

File: qmlui/qlcplus_de_DE.ts [hunk 2]
```diff
@@ -0,0 +1,4150 @@
...
+        <translation>CMY</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="379"/>
+        <source>Add a new color filters file</source>
+        <translation>Neue Farbfilterdatei hinzufügen</translation>
+    </message>
+    <message>
+        <location filename="qml/ColorToolFilters.qml" line="393"/>
+        <source>Rename the current color filters</source>
```
Comment (fxedel) [inline qmlui/qlcplus_de_DE.ts]: @mcallegari Is this actually "Rename the current color filters _file_"? Or does it refer to a single filter and not the filter file?

Comment (mcallegari) [reply qmlui/qlcplus_de_DE.ts]: Yeah, it refers to the file, not a single filter. I need to review that part cause it didn't look right to me either. For the time being, if unsure, just leave the string untranslated.

**Comment by mcallegari** [comment]
@fxedel please let me know when this can be merged

File: qmlui/qlcplus_de_DE.ts [hunk 3]
```diff
@@ -836,614 +836,614 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="108"/>
         <source>Reset the entire group</source>
-        <translation type="unfinished"></translation>
+        <translation>Gesamte Gruppe zurücksetzen</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupEditor.qml" line="131"/>
         <source>Rotate 90° clockwise</source>
-        <translation type="unfinished"></translation>
+        <translation>Um 90° drehen (im UZS)</translation>
```
Comment (FloEdelmann) [inline qmlui/qlcplus_de_DE.ts]: In most desktop applications, it's "Um 90° nach rechts drehen", which sounds a bit nicer.

File: qmlui/qlcplus_de_DE.ts [hunk 4]
```diff
@@ -836,614 +836,614 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/fixturesfunctions/FixtureGroupManager.qml" line="258"/>
         <source>Flags</source>
-        <translation type="unfinished"></translation>
+        <translation>Markierungen</translation>
     </message>
     <message>
         <location filename="qml/fixturesfunctions/FixtureGroupManager.qml" line="260"/>
         <source>Can fade</source>
-        <translation type="unfinished"></translation>
+        <translation>Kann faden</translation>
```
Comment (FloEdelmann) [inline qmlui/qlcplus_de_DE.ts]: "überblenden" statt "faden"?

File: qmlui/qlcplus_de_DE.ts [hunk 5]
```diff
@@ -836,614 +836,614 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/inputoutput/IORightPanel.qml" line="90"/>
         <source>Show the universe output sources</source>
-        <translation type="unfinished"></translation>
+        <translation>Universumsausgangsquellen zeigen</translation>
     </message>
     <message>
         <location filename="qml/inputoutput/IORightPanel.qml" line="107"/>
         <source>Enable/Disable blackout on all the output patches</source>
-        <translation type="unfinished"></translation>
+        <translation>Blackout in alle Ausgangsprofilen ein-/ausschalten</translation>
```
Comment (FloEdelmann) [inline qmlui/qlcplus_de_DE.ts]: alle**n**

File: qmlui/qlcplus_de_DE.ts [hunk 6]
```diff
@@ -1799,50 +1799,50 @@ Ungesicherte Änderungen gehen verloren, wenn du sie nicht speicherst.</translat
...
         <location filename="qml/popup/PopupMonitor.qml" line="49"/>
         <source>Please select the initial point of view for your 2D preview</source>
-        <translation type="unfinished"></translation>
+        <translation>Wähle bitte den ursprünglichen Blickpunkt für deine 2D-Vorschau</translation>
     </message>
     <message>
         <location filename="qml/popup/PopupMonitor.qml" line="65"/>
         <source>Top view</source>
-        <translation type="unfinished"></translation>
+        <translation>Von Oben</translation>
```
Comment (FloEdelmann) [inline qmlui/qlcplus_de_DE.ts]: lowercase "oben"

**Comment by mcallegari** [comment]
@siegmund42  @kripton any chance to help here ?
QLC+ 5 Alpha 3 is nearing and it would be nice to have German translation in. Thanks.

**Comment by siegmund42** [comment]
@mcallegari Do you have a particular date in mind? I'd really like to help, but I'm very busy this week.

**Comment by mcallegari** [comment]
@siegmund42 I'd like to release something within July. Maybe the 28th-29th weekend

**Comment by kripton** [comment]
Hi, sorry that this got under my radar. The regular checks I do it to scan existing .ts files for missing strings. This time, a new .ts-file is needed and I didn't have this case in mind.
I'm currently on tour (also using QLC+ :D) so I cannot work on this this weekend. On Tuesday I'll have plenty of time and will review & complete the translations.
@fxedel: Thanks for the work already done
@siegmund42: Feel free to work on this before, I'll check what is there on Tuesday

**Comment by siegmund42** [comment]
@kripton Sorry I won't be able to work on this by end of July so feel free to move on! But I'll be happy to review the work in August!

**Comment by kripton** [comment]
Should be all done in #1096 :+1:

**Pull request merged**